dockerfile

```diff
@@ -26,7 +26,8 @@ RUN /usr/local/venv/bin/pip install \
     "dbt-redshift==1.3.0" \
     "dbt-snowflake==1.3.0" \
     "dbt-bigquery==1.3.0" \
-    "dbt-databricks==1.3.0"
+    "dbt-databricks==1.3.0" \
+    "dbt-trino==1.3.1"
```

Comment:
We need the same change in the dockerfile-prs so our PR environments can have this adapter to run the E2E tests.